installer/lib/phoenix_new.ex

```diff
@@ -226,6 +227,7 @@ defmodule Mix.Tasks.Phoenix.New do
       # Configure your database
       config :#{binding[:application_name]}, #{binding[:application_module]}.Repo,
         adapter: #{inspect binding[:adapter_module]},
+        url: System.get_env("DATABASE_URL"),
```

Comment:
I checked and found that if url is nil, the regular ones will be used. If url is present and is valid, it will override the regular one.